tests/test_vec_normalize.py

```diff
@@ -76,6 +76,42 @@ def test_vec_env(tmpdir):
     check_vec_norm_equal(norm_venv, deserialized)
 
 
+def _make_stepped_cartpole():
```

Comment:
Why do we need this? To ensure the normalization statistics are "warm-started"? Good to add a docstring explaining this.